packages/fetchai/skills/confirmation_aw3/skill.yaml

```diff
@@ -32,6 +32,11 @@ behaviours:
     args:
       search_interval: 1800
     class_name: SearchBehaviour
+  transaction:
+    args:
+      max_processing: 420
+      transaction_interval: 2
```

Comment:
not sure it's relevant; maybe a comment somewhere on what these parameters do? Don't know what's the best place; maybe in this YAML just above those lines, or in the skill README. We should update the other too though